Security/Http/Authentication/AuthenticationFailureHandler.php

```diff
@@ -35,8 +35,13 @@ public function __construct(EventDispatcherInterface $dispatcher)
      */
     public function onAuthenticationFailure(Request $request, AuthenticationException $exception)
     {
-        $event = new AuthenticationFailureEvent($exception, new JWTAuthenticationFailureResponse());
+        if ('' != $exception->getMessage()) {
```

Comment:
Maybe a dumb question, but:
Any reason to this if?
Because calling the `JWTAuthenticationFailureResponse` constructor without arguments defaults to calling it with empty string which is the condition to call the constructor without argument.